Error during npm test after updating Angular

# 🐞 bug report

### Affected Package
I believe the issue was introduced by one of the updated packages by `ng update @angular/core @angular/cli`.

### Is this a regression?
Yes, the issue was not present in version 11.0.0

### Description
After updating the Angular dependencies of the library `angular-archwizard` from `11.0.x` to latest by executing `ng update @angular/core @angular/cli`, `npm test` fails with the following error message:

```
$ npm test

> angular-archwizard@8.0.0-develop test ~/git/angular-archwizard
> ng test && ng test --tsConfig=tsconfig-noIvy.spec.json

Compiling @angular/compiler/testing : es2015 as esm2015
Compiling @angular/core : es2015 as esm2015
Compiling @angular/common : es2015 as esm2015
Compiling @angular/core/testing : es2015 as esm2015
Compiling @angular/platform-browser : es2015 as esm2015
Compiling @angular/platform-browser-dynamic : es2015 as esm2015
Compiling @angular/platform-browser/testing : es2015 as esm2015
Compiling @angular/platform-browser-dynamic/testing : es2015 as esm2015
⠋ Generating browser application bundles...21 02 2021 18:51:49.235:INFO [framework.detect-browsers]: The following browsers will be used: [ 'FirefoxHeadless' ]
⠙ Generating browser application bundles (phase: building)...21 02 2021 18:51:49.325:INFO [karma-server]: Karma v6.1.1 server started at http://localhost:9876/
21 02 2021 18:51:49.325:INFO [launcher]: Launching browsers FirefoxHeadless with concurrency unlimited
21 02 2021 18:51:49.327:INFO [launcher]: Starting browser FirefoxHeadless
✔ Browser application bundle generation complete.
21 02 2021 18:51:53.859:INFO [Firefox 85.0 (Linux x86_64)]: Connected on socket xICXANRpTidBS4xtAAAB with id 42976427
Firefox 85.0 (Linux x86_64) ERROR
  An error was thrown in afterAll
  ReferenceError: undefinedundefinedundefinedundefinedundefined is not defined
  ierq@http://localhost:9876/_karma_webpack_/main.js:5603:1
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  HACE@http://localhost:9876/_karma_webpack_/main.js:3229:100
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  uFF0@http://localhost:9876/_karma_webpack_/main.js:6561:109
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  llYR@http://localhost:9876/_karma_webpack_/main.js:5709:95
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  webpackContext@http://localhost:9876/_karma_webpack_/main.js:5298:9
  CB3P@http://localhost:9876/_karma_webpack_/main.js:2095:16
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  checkDeferredModules@http://localhost:9876/_karma_webpack_/main.js:46:23
  @http://localhost:9876/_karma_webpack_/main.js:153:18
  @http://localhost:9876/_karma_webpack_/main.js:156:10
Firefox 85.0 (Linux x86_64): Executed 0 of 0 ERROR (0.042 secs / 0 secs)
npm ERR! Test failed.  See above for more details.
```

## 🔬 Minimal Reproduction
1. clone `angular-archwizard` from https://github.com/madoar/angular-archwizard
2. install all required dependencies (`npm install`)
3. update all Angular dependencies (`ng update @angular/core @angular/cli`)
4. execute the tests `npm test`
5. see above error message in the terminal

## 🔥 Exception or Error
```
21 02 2021 18:51:53.859:INFO [Firefox 85.0 (Linux x86_64)]: Connected on socket xICXANRpTidBS4xtAAAB with id 42976427
Firefox 85.0 (Linux x86_64) ERROR
  An error was thrown in afterAll
  ReferenceError: undefinedundefinedundefinedundefinedundefined is not defined
  ierq@http://localhost:9876/_karma_webpack_/main.js:5603:1
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  HACE@http://localhost:9876/_karma_webpack_/main.js:3229:100
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  uFF0@http://localhost:9876/_karma_webpack_/main.js:6561:109
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  llYR@http://localhost:9876/_karma_webpack_/main.js:5709:95
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  webpackContext@http://localhost:9876/_karma_webpack_/main.js:5298:9
  CB3P@http://localhost:9876/_karma_webpack_/main.js:2095:16
  __webpack_require__@http://localhost:9876/_karma_webpack_/main.js:80:30
  checkDeferredModules@http://localhost:9876/_karma_webpack_/main.js:46:23
  @http://localhost:9876/_karma_webpack_/main.js:153:18
  @http://localhost:9876/_karma_webpack_/main.js:156:10
Firefox 85.0 (Linux x86_64): Executed 0 of 0 ERROR (0.042 secs / 0 secs)
npm ERR! Test failed.  See above for more details.
```

## 🌍  Your Environment
```
$ ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 11.2.1
Node: 14.15.5
OS: linux x64

Angular: 11.2.1
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1102.1
@angular-devkit/build-angular   0.1102.1
@angular-devkit/core            11.2.1
@angular-devkit/schematics      11.2.1
@schematics/angular             11.2.1
@schematics/update              0.1102.1
rxjs                            6.6.3
typescript                      4.0.7
```

**Anything else relevant?**
For the tests `Firefox` has been used. I didn't test whether this issue also occurs with a different browser.

--- a/packages/angular_devkit/build_angular/src/webpack/configs/typescript.ts
+++ b/packages/angular_devkit/build_angular/src/webpack/configs/typescript.ts
@@ -54,6 +54,8 @@ function createIvyPlugin(
 
   const compilerOptions: CompilerOptions = {
     sourceMap: buildOptions.sourceMap.scripts,
+    declaration: false,
+    declarationMap: false,
   };
 
   if (buildOptions.preserveSymlinks !== undefined) {

